### PR TITLE
Add a react-experimental library

### PR DESCRIPTION
**Pull request by rictic**
Shows the current status of react@experimental – 100% passing tests!

### CODE REVIEW

File: libraries/react-experimental/src/basic-tests.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+      expect(data).to.eql("React");
+    });
+
+    // TODO: Is it the framework's responsibility to check if the underlying
+    // property is defined? Or should it just always assume it is and do its
+    // usual default behavior? Preact will actually check if it's defined and
+    // use an attribute if it is not, otherwise it prefers properties for
+    // everything. Is there a "right" answer in this situation?
+
+    // it('will set boolean attributes on a Custom Element that has not already been defined and upgraded', function() {
```
Comment (dfreedm) [inline libraries/react-experimental/src/basic-tests.js:135]: What about using `xit` rather than commenting these out?

**Review by dfreedm — CHANGES_REQUESTED** [review]

File: libraries/react-experimental/src/basic-tests.js (hunk 1 above)
Comment (dfreedm) [reply libraries/react-experimental/src/basic-tests.js:135]: Oh wait, is this still `mocha` under the hood? You might have to use `this.skip()` in the test instead.

Comment (rictic) [reply libraries/react-experimental/src/basic-tests.js:135]: These tests are commented out in all the repos, will delete in a follow up

**Review by dfreedm — APPROVED** [review]

**Comment by dfreedm** [comment]
LGTM

**Pull request merged**